MetaMask/metamask-extension
# UX Multichain: Updated account picker in new send flow

This PR is to update the account picker in Send Flow

Fixes: [#1581](https://github.com/MetaMask/MetaMask-planning/issues/1581)

## **Manual testing steps**

1. Go to Send Flow in Multichain Flow
2. Check the account picker matches the figma

## **Screenshots/Recordings**


### **Before**
![Screenshot 2023-10-27 at 2 43 15 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/634fd28e-2816-4cf9-8f4c-c42340446fd1)


### **After**
![Screenshot 2023-10-27 at 2 42 10 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/7c557a3c-a331-4b1e-8361-18c64d42cca9)


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
